.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,43 @@
+name: Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      next-version:
+        description: 'Override next development version (e.g. 3.0.0-SNAPSHOT). If not set then the version will be calculated by maven release plugin'
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    name: Prepare release
+    permissions:
+          contents: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ssh-key: "${{ secrets.COMMIT_KEY }}"
+      - name: Setup JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version:  17
+          distribution: 'temurin'
+          cache: 'maven'
+          gpg-private-key: ${{ secrets.MAVEN_GPG_PRIVATE_KEY }}
+          gpg-passphrase: MAVEN_GPG_PASSPHRASE
+
+      - name: Prepare release
+        run: |
+          if [ -n "${{ github.event.inputs.next-version }}" ]; then
+              EXTRA_ARGS=-DdevelopmentVersion=${{ github.event.inputs.next-version }}
+          fi
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+          ./mvnw --batch-mode release:prepare -Prelease -Darguments="-DskipTests" $EXTRA_ARGS
```

Comment:
we do not have mvnw in this repo

Reply:
Added https://github.com/hazelcast/quarkus-hazelcast-client/pull/357/commits/c184b5ab142d308be73a293e38e96b64eb9c29d4